src/devices/mazda.ts

```diff
@@ -1,9 +1,15 @@
+import {KeyValue} from 'zigbee-herdsman/dist/controller/tstype';
```

Comment:
Please use this one instead:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/8f313c3643b07fbba2d7a2e7447293590ee0f7e7/src/lib/types.ts#L17